tests/InternalClient.php

```diff
@@ -160,6 +165,15 @@ public function getUserID() {
         return $this->userID;
     }
 
+    /**
+     * Get the user ID that will be used to make requests.
+     *
+     * @return int Returns the activityName.
+     */
+    public function getActivityName() {
```

Comment:
Where is this getting set? In any case it definitely shouldn't be part of our Http client.